doc/cli/operator-sdk_run_helm.md

```diff
@@ -15,6 +15,7 @@ operator-sdk run helm [flags]
 ### Options
 
 ```
+      --ansible-roles-path string        Ansible path roles ($ANSIBLE_ROLES_PATH). By default it will be /roles in the current directory
```

Comment:
I don't think this option should be in run_helm.md right?

Reply:
It is gen automatically. I will see how to fix.

Reply:
Solved. Good spot.